Re-sync WPT's resources/testdriver.js

lukewarlow opened the pull request:
#### 7ae49b1e2fa6471627ca69cc04b47b630666f12e
<pre>
Re-sync WPT&apos;s resources/testdriver.js
<a href="https://bugs.webkit.org/show_bug.cgi?id=274621">https://bugs.webkit.org/show_bug.cgi?id=274621</a>

Reviewed by Tim Nguyen.

Upstream Commit: <a href="https://github.com/web-platform-tests/wpt/commit/1a6206c84228a17ab23a7e8b02768d1f15655705">https://github.com/web-platform-tests/wpt/commit/1a6206c84228a17ab23a7e8b02768d1f15655705</a>

* LayoutTests/imported/w3c/web-platform-tests/resources/testdriver.js:
(window.test_driver.send_keys):
(window.test_driver.get_window_rect):
(window.test_driver.set_rph_registration_mode):
(window.test_driver.cancel_fedcm_dialog):
(window.test_driver.click_fedcm_dialog_button):
(window.test_driver.select_fedcm_account):
(window.test_driver.get_fedcm_account_list):
(window.test_driver.get_fedcm_dialog_title):
(window.test_driver.get_fedcm_dialog_type):
(window.test_driver.set_fedcm_delay_enabled):
(window.test_driver.reset_fedcm_cooldown):
(window.test_driver.create_virtual_sensor):
(window.test_driver.update_virtual_sensor):
(window.test_driver.remove_virtual_sensor):
(window.test_driver.get_virtual_sensor_information):
(window.test_driver.set_device_posture):
(window.test_driver.clear_device_posture):
(window.test_driver_internal.async get_window_rect):
(window.test_driver_internal.set_rph_registration_mode):
(window.test_driver_internal.async cancel_fedcm_dialog):
(window.test_driver_internal.async click_fedcm_dialog_button):
(window.test_driver_internal.async select_fedcm_account):
(window.test_driver_internal.async get_fedcm_account_list):
(window.test_driver_internal.async get_fedcm_dialog_title):
(window.test_driver_internal.async get_fedcm_dialog_type):
(window.test_driver_internal.async set_fedcm_delay_enabled):
(window.test_driver_internal.async reset_fedcm_cooldown):
(window.test_driver_internal.async create_virtual_sensor):
(window.test_driver_internal.async update_virtual_sensor):
(window.test_driver_internal.async remove_virtual_sensor):
(window.test_driver_internal.async get_virtual_sensor_information):
(window.test_driver_internal.async set_device_posture):
(window.test_driver_internal.async clear_device_posture):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-change-type-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-close-request-expected.txt:

Canonical link: <a href="https://commits.webkit.org/279320@main">https://commits.webkit.org/279320@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/035a8910f5428c13cb10c46f49b897615dc923b6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/53135 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/32472 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/55/builds/5622 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/56414 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/59/builds/3858 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/39366 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/3587 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/43078 "Passed tests") | [✅ 🧪 wincairo-tests](https://ews-build.webkit.org/#/builders/60/builds/2499 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/55233 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/30244 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/45865 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/24208 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/27238 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/64/builds/3193 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/2017 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/49089 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/63/builds/3351 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/58010 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/28276 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/62/builds/3297 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/50481 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/29496 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/46091 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/49787 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/11592 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/30415 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/29250 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m reviewed (commented):
r=me assuming green tests

lukewarlow committed:
Re-sync WPT's resources/testdriver.js
https://bugs.webkit.org/show_bug.cgi?id=274621

Reviewed by Tim Nguyen.

Upstream Commit: https://github.com/web-platform-tests/wpt/commit/1a6206c84228a17ab23a7e8b02768d1f15655705

* LayoutTests/imported/w3c/web-platform-tests/resources/testdriver.js:
(window.test_driver.send_keys):
(window.test_driver.get_window_rect):
(window.test_driver.set_rph_registration_mode):
(window.test_driver.cancel_fedcm_dialog):
(window.test_driver.click_fedcm_dialog_button):
(window.test_driver.select_fedcm_account):
(window.test_driver.get_fedcm_account_list):
(window.test_driver.get_fedcm_dialog_title):
(window.test_driver.get_fedcm_dialog_type):
(window.test_driver.set_fedcm_delay_enabled):
(window.test_driver.reset_fedcm_cooldown):
(window.test_driver.create_virtual_sensor):
(window.test_driver.update_virtual_sensor):
(window.test_driver.remove_virtual_sensor):
(window.test_driver.get_virtual_sensor_information):
(window.test_driver.set_device_posture):
(window.test_driver.clear_device_posture):
(window.test_driver_internal.async get_window_rect):
(window.test_driver_internal.set_rph_registration_mode):
(window.test_driver_internal.async cancel_fedcm_dialog):
(window.test_driver_internal.async click_fedcm_dialog_button):
(window.test_driver_internal.async select_fedcm_account):
(window.test_driver_internal.async get_fedcm_account_list):
(window.test_driver_internal.async get_fedcm_dialog_title):
(window.test_driver_internal.async get_fedcm_dialog_type):
(window.test_driver_internal.async set_fedcm_delay_enabled):
(window.test_driver_internal.async reset_fedcm_cooldown):
(window.test_driver_internal.async create_virtual_sensor):
(window.test_driver_internal.async update_virtual_sensor):
(window.test_driver_internal.async remove_virtual_sensor):
(window.test_driver_internal.async get_virtual_sensor_information):
(window.test_driver_internal.async set_device_posture):
(window.test_driver_internal.async clear_device_posture):
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-change-type-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/html/semantics/popovers/popover-close-request-expected.txt:

Canonical link: https://commits.webkit.org/279320@main
